src/views/login/login.view.tsx

```diff
@@ -36,49 +38,21 @@ const Login: FC = () => {
         Bridge
       </Typography>
       <Card className={classes.card}>
-        <Typography type="h1" className={classes.cardHeader}>
-          Connect a wallet
-        </Typography>
-        <ul className={classes.walletList}>
-          <li
-            className={classes.wallet}
-            role="button"
-            onClick={() => onConnectProvider(WalletName.METAMASK)}
-          >
-            <div className={classes.walletInfo}>
-              <div className={`${classes.walletIconContainer} ${classes.metaMaskIconContainer}`}>
-                <MetaMaskIcon />
-              </div>
-              <div>
-                <Typography type="body1" className={classes.walletName}>
-                  MetaMask
-                </Typography>
-                <Typography type="body2">Connect using web wallet</Typography>
-              </div>
-            </div>
-            <CaretRightIcon />
-          </li>
-          <li
-            className={classes.wallet}
-            role="button"
-            onClick={() => onConnectProvider(WalletName.WALLET_CONNECT)}
-          >
-            <div className={classes.walletInfo}>
-              <div
-                className={`${classes.walletIconContainer} ${classes.walletConnectIconContainer}`}
-              >
-                <WalletConnectIcon />
-              </div>
-              <div>
-                <Typography type="body1" className={classes.walletName}>
-                  WalletConnect
-                </Typography>
-                <Typography type="body2">Connect using mobile wallet</Typography>
-              </div>
-            </div>
-            <CaretRightIcon />
-          </li>
-        </ul>
+        {account.status === "pending" || account.status === "failed" ? (
```

Comment:
Here I'm missing `|| selectedWallet === undefined`

Reply:
The problem is that I wasn't resetting the `selectedWallet` to `undefined` in any case (which is wrong). I've uploaded a different approach. I think that the code looks cleaner and we depend only on the local state of the view to render it.

Reply:
Beautiful